scripts/lst-recopipe.py

```diff
@@ -63,10 +63,8 @@
     #dl1_file=args.datafile
     #data = pd.read_hdf(args.datafile, key='events/LSTCam')
     intensity_min = np.log10(50)
-    r_max = 10#0.8
 
-    data = dl1_to_dl2.filter_events(pd.read_hdf(dl1_file, key='events/LSTCam'), 
-                             r_max=r_max, intensity_min=intensity_min)
+    data = dl1_to_dl2.filter_events(pd.read_hdf(dl1_file, key='events/LSTCam'), intensity_min=intensity_min)
```

Comment:
`r_max` was the maximum impact parameter or the Hillas parameter `r`? If it is the former, it may be useful to apply cuts on it to perform tests, but from its previous value it does not look like

Reply:
Hi @rlopezcoto , r is the hillas parameter (distance between center of camera and c.o.g. of the image). I was using cuts in r_max before we had the leakage parameter, to avoid showers in the border of the camera. Now it makes no sense to do cuts in r_max anymore.

Reply:
`r_max` is not an option of the dl1_to_dl2 reconstruction anymore. I think it @misabelber removed it in some previous PR but it still appeared in some functions. I just fix the code for the functions to work here.